### PR TITLE
Update form entry to display lab order-numbers

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.

## Summary

- This PR updates the form-entry-app to display order numbers when entry of data is completed
- Updated the edit and cancel button to match carbon themed buttons

## Screenshots

Screen Recording 2022-01-06 at 15 11 12

## Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Review by jonathandick — APPROVED** [review]
LGTM

**Comment by gracepotma** [comment]
FYI @enyachoke

File: packages/esm-form-entry-app/src/app/form-submitted/form-submitted.component.html [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-            Edit Form
-        </a>
-        <a class="omrs-link omrs-outlined-action" (click)="cancel()">
-            Close
-        </a>
-    </div>
+<div class="form-submission-container">
+  <div class="success-message">
+      <span>The form was successfully submitted.</span> 
+      <svg class="omrs-icon" fill="var(--omrs-color-success)"
```
Comment (enyachoke) [inline packages/esm-form-entry-app/src/app/form-submitted/form-submitted.component.html:4]: Should we maybe update this to match other system notifications if possible?

**Review by enyachoke — COMMENTED** [review]
I have made a comment about the success message otherwise LGTM

File: packages/esm-form-entry-app/src/app/form-submitted/form-submitted.component.html (hunk 1 above)
Comment (donaldkibet) [reply packages/esm-form-entry-app/src/app/form-submitted/form-submitted.component.html]: @enyachoke i have updated to be consistent with the way other forms are save here is a recording

https://user-images.githubusercontent.com/28008754/148534532-b411ad85-b3d3-410e-ab95-b9132ddd083c.mov

**Review by enyachoke — APPROVED** [review]

**Pull request merged**